.github/workflows/miri.yml

```diff
@@ -11,7 +11,6 @@ jobs:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != 'amazon-ion/ion-rust'
     strategy:
       matrix:
-        # See https://github.com/amazon-ion/ion-rust/issues/353
```

Comment:
🗺️ Issue #353 was fixed in #688, which changed the CI task back to `windows-latest`.